pkg/apiserver/rest/usecase/project.go

```diff
@@ -613,3 +617,54 @@ func retrieveConfigFromApplication(a v1beta1.Application, project string) *apisv
 		Description:       a.Annotations[types.AnnotationConfigDescription],
 	}
 }
+
+func (p *projectUsecaseImpl) ValidateImage(ctx context.Context, projectName, image string) (*apisv1.ImageResponse, error) {
+	return validateImage(ctx, p.k8sClient, projectName, image)
+}
+
+func validateImage(ctx context.Context, k8sClient client.Client, project, imageName string) (*apisv1.ImageResponse, error) {
```

Comment:
Please add test cases for private repositories, including the docker hub.

Reply:
Done, thanks.